scipy/integrate/quadpack.py

```diff
@@ -102,9 +102,14 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
     Other Parameters
     ----------------
     epsabs : float or int, optional
-        Absolute error tolerance.
+        Absolute error tolerance. Default is 1.49e-8. Quad tries to obtain
+        an accuracy of `abs(i-result) <= max(epsabs, epsrel*abs(i))`
```

Comment:
A note on reST formatting: typesetting as code requires double backticks (single backticks means "create a link to this function/object").